src/lib/tracking/withTracking.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import track from "./track";
+import dispatch from "./dispatch";
+
+/**
+ * withTracking higher order component for sending tracking data to centralized dispatch method
+ * @name withTracking
+ * @param {React.Component} Component to decorate and return
+ * @returns {React.Component} - Decorated component
+ */
+export default (Component) => (
+  @track({}, { dispatch })
+  class withTracking extends React.Component {
+    render() {
+      return (
+        <Component {...this.props} />
+      );
+    }
+  }
+);
```

Comment:
To have fewer unnecessary layers, I'd prefer to use the NYT recommended approach of passing dispatch just once, in the top layer of the app: https://github.com/NYTimes/react-tracking#custom-optionsdispatch-for-tracking-data

Then `withTracking` should be unnecessary.

Was there a reason this approach doesn't work?